bot.py

```diff
@@ -46,9 +82,44 @@ async def ping(interaction: discord.Interaction):
     name="register", description="Register as a hacker for matchmaking."
 )
 async def register(interaction: discord.Interaction):
+    structure_roles = [
```

Comment:
Before the hackathon start make sure to sweep this again to make sure the roles are updated